app.py

```diff
@@ -119,12 +119,33 @@ def auth_granted():
     @app.route('/api/slack/commands', methods=['POST'])
     def slash_command():
         data = request.form.to_dict()
+        print('data: ', data)
 
         if data.get('type'):
-            return(get_challenge_response(data, app.config['SLACK_VERIFICATION_TOKEN'])) 
+            return(get_challenge_response(data, app.config['SLACK_VERIFICATION_TOKEN']))
         else:
-            return(jsonify({ "text": "Test reply: " + data['text'] + " from " + data['user_name'] }))
-
+            # /api/slack/commands with no params - give top 5 received emojis for current user
+            if(data.get('text') == ''):
```

Comment:
use `if data.get('text').isSpace():` to handle this if there happens to be whitespace

[stackoverflow](https://stackoverflow.com/questions/2405292/how-to-check-if-text-is-empty-spaces-tabs-newlines-in-python)